app/views/dashboard/index.html.erb

```diff
@@ -0,0 +1,20 @@
+<h3 class="text-center"> Top Three Companies Average Level of interest </h3>
+<div class="text-center">
+  <%= link_to "Jobs Sorted by Interest", ("#{jobs_path}""?sort=interest"), class: "btn btn-primary" %>
+  <%= link_to "Jobs Sorted by Location", ("#{jobs_path}""?sort=location"), class: "btn btn-success" %>
+</div>
+<% @company.each do |company, number| %>
+<li class="text-center">
+  <strong><%= "#{Company.find(company).name}'s jobs have a #{number} average level of Interest"%></strong>
```

Comment:
be sure to keep your db queries out of your view. it's the responsibility of the controller to grab your data and the responsibility of your view to simply display the data